lms/djangoapps/course_wiki/course_app.py

```diff
@@ -0,0 +1,64 @@
+"""Module with the course app configuration for the Wiki."""
+from typing import Dict, Optional
+
+from django.contrib.auth import get_user_model
+from django.utils.translation import ugettext_noop as _
+from edx_toggles.toggles import SettingToggle
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.djangoapps.course_apps.plugin import CourseApp
+
+User = get_user_model()
+
+
+class WikiCourseApp(CourseApp):
+    """
+    Course app for the Wiki.
+    """
+
+    app_id = "wiki"
+    name = _("Wiki")
+    description = _("Enable learners to access, and collaborate on information about your course.")
+    default_enabled = SettingToggle(name="WIKI_ENABLED", default=True).is_enabled()
+
+    @classmethod
+    def is_available(cls, course_key: CourseKey) -> bool:  # pylint: disable=unused-argument
+        """
+        Returns if the app is available for the course.
+
+        The wiki is available for all courses or none of them depending on the a Django setting.
```

Comment:
The wiki is available for all courses? It cannot be be made available for a particular course?